tests/nnstreamer_sink/unittest_sink.cpp

```diff
@@ -739,58 +740,65 @@ _setup_pipeline (TestOption & option)
       /** 4x4 tensor stream, different FPS, tensor_mux them @ slowest */
       str_pipeline =
           g_strdup_printf
-	  ("videotestsrc pattern=snow num-buffers=%d ! video/x-raw,format=BGRx,height=4,width=4,framerate=10/1 ! tensor_converter ! tensor_filter framework=custom model=./tests/libnnscustom_framecounter.so ! mux.sink_0 "
-	  "videotestsrc pattern=snow num-buffers=%d ! video/x-raw,format=BGRx,height=4,width=4,framerate=25/1 ! tensor_converter ! tensor_filter framework=custom model=./tests/libnnscustom_framecounter.so ! mux.sink_1 "
-	  "tensor_mux sync_mode=slowest name=mux ! tensor_filter framework=custom model=./tests/libnnscustom_framecounter.so ! tee name=t ! queue ! tensor_sink sync=true name=test_sink t. ! queue ! filesink location=%s", option.num_buffers * 10, option.num_buffers * 25, option.tmpfile);
+          ("videotestsrc pattern=snow num-buffers=%d ! video/x-raw,format=BGRx,height=4,width=4,framerate=10/1 ! tensor_converter ! tensor_filter framework=custom model=./tests/libnnscustom_framecounter.so ! mux.sink_0 "
```

Comment:
BTW, is this change  related to the goal of this PR? 
What for? for code readability? 

If you want to changes more than two in one commit, I wan you to use  the below template in the commit body. 

For example, 

```
1. Commit title (ASAP within 53 characters)
2. Commit body (describe a goal in detail  ASAP)
3. If the commit is +two goal, use **Changes proposed in this PR:**
```

Reply:
Be careful. gst-indent is not compatible with C++.

Reply:
So. I am going to enable CPPcheck to control CPP source code. Could you give me opinions?
@ohsewon  PTAL.

Reply:
I made mistake. In case of NNStreamer, @ohsewon already enabled the module "CPPcheck".  :)